dali/pipeline/operators/crop/basic/crop.h

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Can't we reuse current crop code? Do we really want to reimplement crop?
Also if we use nvvl we need GPU crop for the sequences.